File: FoodDelivery_Backend/Models/SupplierType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FoodDelivery_Backend.Models
{
    public class SupplierType
    {
        public string supplier_type { get; set; }
        public string description { get; set; }
        public decimal yearly_turnover { get; set; }

    }
}

File: FoodDelivery_Angular_Frontend/Frontend-App/src/app/Models/supplierType.ts

export class supplierTypeModel {
    supplier_type: string;
    description: string;
    yearly_turnover: number;
    isEdit: boolean;


    constructor() {
        this.supplier_type = '';
        this.description = '';
        this.yearly_turnover = 0;
        this.isEdit=false;
    }
}
